frontends/verific/verific.cc

```diff
@@ -199,12 +199,17 @@ void VerificImporter::import_attributes(dict<RTLIL::IdString, RTLIL::Const> &att
 				attributes.emplace(stringf("\\enum_value_%s", p+2), RTLIL::escape_id(k));
 			}
 			else if (nl->IsFromVhdl()) {
-				// Expect "<binary>"
+				// Expect "<binary>" or plain <binary>
 				auto p = v;
 				if (p) {
-					if (*p != '"')
-						p = nullptr;
-					else {
+					if (*p != '"') {
+						auto l = strlen(p);
+						auto q = (char*)malloc(l+1);
+						strncpy(q, p, l);
+						q[l] = '\0';
+						for(char *ptr = q; *ptr; ++ptr )*ptr = tolower(*ptr);
+						attributes.emplace(stringf("\\enum_value_%s", q), RTLIL::escape_id(k));
+					} else {
 						auto *q = p+1;
 						for (; *q != '"'; q++)
 							if (*q != '0' && *q != '1') {
```

Comment:
This looks to me like we allow `x` but not `"x"`. I think we should not have an inconsistency like that.